modules/update-upgrading-web.adoc

```diff
@@ -85,7 +85,7 @@ ifdef::rhel[]
 +
 [NOTE]
 ====
-When you update a cluster that contains Red Hat Enterprise Linux (RHEL) worker machines, those workers temporarily become unavailable during the update process. You must run the update playbook against each RHEL machine as it enters the `NotReady` state for the cluster to finish updating.
+When you update a cluster that contains {op-system-base-full} worker machines, those workers temporarily become unavailable during the update process. You must run the update playbook against each {op-system-base} machine as it enters the `NotReady` state for the cluster to finish updating.
```

Comment:
In this case we're using "you must" to indicate that it's something you have to do, however it's not exactly written as a command (for instance, using the imperative mood if you were to say "run the update playbook against each {op-system-base} machine as it enters the `NotReady` state for the cluster to finish updating.")

If it's okay with you, I'd prefer to keep it as is because the way it's written helps describe the process as a requirement.

That said, if this is critical to the accomplishment of the task, I think I'll change this from a NOTE to IMPORTANT

Reply:
Thanks! keep it as is sounds good.